src/main/java/irwyn/tasks/Task.java

```diff
@@ -0,0 +1,94 @@
+package irwyn.tasks;
+
+/**
+ * This class encapsulates the Task class.
+ * It represents a Task.
+ *
+ * @author Irwyn Liong
+ * @version Week-3
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    TaskType task;
+
+    public enum TaskType {
+        TODO,
+        EVENT,
+        DEADLINE
+    }
```

Comment:
It is good that your constant variable name is all uppercase and is clear to other people.